src/data.ts

```diff
@@ -0,0 +1,30 @@
+export const datos = {
+    contactInfo: {
+        fullName: "Danilo Antonio Solórzano Rodas",
+        emailAddress: "danilo_solorzano06@hotmail.com",
+    },
+    github: {
+        profileURL: "https://github.com/danbart",
+        username: "danbart",
+    },
+    credentials: {
+        password: "",
+    },
+personalInfo: {
+    questions: [
+        {
+            question: "If I was a Sr. Programmer, I would like to build:",
+            answer: "Respuesta: Una plataforma educativa que ayude a las persones médiate interacciones a mejorar su ortografía de una forma divertida, que esta no sirva para corregir textos si no que la gente aprenda las reglas gramaticales y ortográficas.",
```

Comment:
Empecemos por corregir nuestras propias faltas:

```suggestion
            answer: "Respuesta: Una plataforma educativa que ayude a las personas mediante interacciones a mejorar su ortografía de una forma divertida, que esta no sirva para corregir textos si no que la gente aprenda las reglas gramaticales y ortográficas.",
```

Es importante poner atención a los detalles.